explorer/src/providers/accounts/flagged-accounts.tsx

```diff
@@ -1,31 +1,62 @@
 import React from "react";
-import { fetch } from "cross-fetch";
+import { Link } from "react-router-dom";
 
-const FLAGGED_REGISTRY =
```

Comment:
I presume this was here so that we could ‘quickly’ add accounts to [the list](https://github.com/solana-labs/solana-flagged-accounts) without a code push?

If Explorer is literally the only thing that uses this list, then this looks fine to me. If there are _other_ people using that list, then we probably shouldn't fork it.

Reply:
Hasn't been touched in 2 years, for now I'm going to just ignore